WORKSPACE

```diff
@@ -79,7 +79,7 @@ http_archive(
 
 load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "npm_install")
 
-node_repositories(node_version = "16.7.0")
+node_repositories(node_version = "16.6.2")
```

Comment:
Why is the version lower than before?

Reply:
I took the latest supported version from the master branch of rules_nodejs, but later realized that we are using rules_nodejs 0.38.0, in which node 16.6.2 is the latest supported one.

Reply:
Have you tried updaing rules_nodejs instead? If that can be done without much hassle I'd do that, older versions tend to become not supported/compatible with the latest Bazel.